src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,155 @@
+    }
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+        actualuser = new User();
+        actualuser.setLogin("sidorov");
+        actualuser.setPassword("123456");
+        actualuser.setAge(35);
+    }
+
+    @Test
+    void age_18_Ok() {
+        actualuser.setAge(18);
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
+    }
+
+    @Test
+    void age_more18_Ok() {
+        actualuser.setAge(29);
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
+    }
+
+    @Test
+    void age_less18_notOk() {
+        actualuser.setAge(17);
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void age_null_notOk() {
+        actualuser.setAge(null);
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void age_0_notOk() {
+        actualuser.setAge(0);
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void age_less0_notOk() {
+        actualuser.setAge(-25);
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void age_maxint_Ok() {
+        actualuser.setAge(2147483647);
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
+    }
+
+    @Test
+    void loginLength_6_Ok() {
+        actualuser.setLogin("123456");
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
+    }
+
+    @Test
+    void loginLength_more6_Ok() {
+        actualuser.setLogin("12345678009");
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
+    }
+
+    @Test
+    void loginLength_less6_notOk() {
+        actualuser.setLogin("12345");
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void loginLength_0_notOk() {
+        actualuser.setLogin("");
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
```

Comment:
Rename the test method `loginLength_0_notOk` to `register_emptyLogin_notOk` to comply with the naming convention.